### PR TITLE
Feature/install api

### PR DESCRIPTION
**Pull request by CarlaCosenza**
Adding route for installing published app

### CODE REVIEW

File: node/clients/billing.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { AppClient, InstanceOptions, IOContext } from '@vtex/api'
+import { GraphQlError } from '../util/errors'
+
+interface InstallResponse {
+  code: string
+  billingOptions?: string
+}
+
+export default class Billing2 extends AppClient {
```
Comment (tlgimenes) [inline node/clients/billing.ts:9]: Why billing2 ?

File: node/index.ts [hunk 2]
```diff
@@ -52,6 +53,10 @@ export default new Service<Clients, State, ParamsContext>({
   clients,
   routes: {
     // `status` is the route ID from service.json. It maps to an array of middlewares (or a single handler).
+    install: method({
+      DEFAULT: methodNotAllowed,
+      GET: [installApp],
```
Comment (tlgimenes) [inline node/index.ts:58]: This GET changes the state ?

File: node/middlewares/installApp.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    vtex: {
+      route: { params },
+    },
+  } = ctx
+  const appID = params.code as string
+  const { name } = parseAppId(appID)
+
+  let versions: RegistryAppVersionsList = {data:[]}
+  try {
+    versions = await ctx.clients.registry.listVersionsByApp(name)
```
Comment (tlgimenes) [inline node/middlewares/installApp.ts:25]: It's better to try..catch then listing

File: node/clients/billing.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      install(appName:"${appName}", termsOfUseAccepted:${termsOfUseAccepted}, force:${force}) {
+        code
+        billingOptions
+      }
+    }`
+    const {
+      data: { data, errors },
+    } = await this.http.postRaw<any>(`/_v/graphql`, { query: graphQLQuery })
+    if (errors) {
+      if (errors.length === 1 && errors[0].extensions?.exception?.response?.data) {
```
Comment (iaronaraujo) [inline node/clients/billing.ts:29]: Is this. `if` necessary? Couldn't you just reuse `GraphQlError(errors)`?

**Pull request merged**